src/appier_extras/parts/admin/models/base.py

```diff
@@ -533,6 +533,11 @@ def disable_s(self):
         self.enabled = False
         self.save()
 
+    def update_meta_s(self, meta):

```

Comment:
move this bellow touch_s and make sure that the provided meta value is a dictionary (eg: `appier.verify(isinstance(meta, dict))`)